SPLASH/src/game/states/GameState.cpp

```diff
@@ -716,6 +711,11 @@ void GameState::updatePerFrameComponentSystems(float dt, float alpha) {
 	m_componentSystems.animationSystem->updatePerFrame();
 	m_componentSystems.audioSystem->update(m_cam, dt, alpha);
 	m_componentSystems.octreeAddRemoverSystem->updatePerFrame(dt);
+
+	// Will probably need to be called last
+	m_componentSystems.entityAdderSystem->update();
+	m_componentSystems.entityRemovalSystem->update();
+	m_componentSystems.octreeAddRemoverSystem->update(dt);
```

Comment:
octreeAddRemoverSystem has a separate updatePerFrame-function, probably does not need to be moved to updatePerFrameComponentSystems